tinydoor/templates/pages/home.html

```diff
@@ -34,6 +45,7 @@ <h1 class="display-4">Measure User Satisfaction</h1>
     </div>
     <!--end of container-->
   <!--end of section-->
+>>>>>>> c7b73082a79eef302b7168ff3e80dc3c2293398b
```

Comment:
Lines 11-48 shows that we have another merge conflict. When I ran this on ```localhost```, the upload button on your branch is not as stylish as the one on the ```master```. 

Did you experience any error with the version of the upload button currently on master @uyennguyen16900? I think if it's possible we should just keep the upload button as is, as it will look better.

Reply:
I'm sure why we have a merge conflict here since I haven't changed anything in html files.

Reply:
Oh it's no problem. The conflict is probably because you just might not have pulled from the ```master``` branch in a while, because I'm assuming this branch was created a while back, before integrated the Bootstrap theme into the HTML.